scripts/setup-fix.bat

```diff
@@ -0,0 +1,5 @@
+( 	where python >null
+	echo Python environment found..
```

Comment:
In this case Python environment found will be printed regardless.

Reply:
```python > null``` and ```found.```

Reply:
> `python > null` and `found.`

cool. done with the changes.